canvasapis.py

```diff
@@ -0,0 +1,150 @@
+import logging
+import requests
+import urllib
+
+
+class GroupsForSections:
+    """
+    This class makes the api calls to create groups for sections. TODO: this class will be taken out into a separate
+    module in a different repo
+    """
+    HTTP_METHOD_POST = 'Post'
+    HTTP_METHOD_GET = 'Get'
+    AUTHORIZATION = 'Authorization'
+    BEARER = 'Bearer '
+    MIME_TYPE_JSON = 'application/json'
+    CONTENT_TYPE = 'Content-type'
+
+    def __init__(self, canvas_token, canvas_url):
+        self.canvas_token = canvas_token
+        self.canvas_url = canvas_url
+
+    def create_group_category(self, group_category_name, course_id):
+        """
+        Creates a group category with a given name for a particular course, In case of exceptions it throws it to
+        calling method
+
+        :param String group_category_name:
```

Comment:
Frankly, i documented for myself so that i don't get confused what type the parameters  when i look back after some time